<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>619</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Phone Book</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QSplitter" name="splitter_6">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QSplitter" name="splitter_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Number:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="numberLine">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="cursor">
            <cursorShape>IBeamCursor</cursorShape>
           </property>
           <property name="whatsThis">
            <string>Number Line</string>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="placeholderText">
            <string>8(XXX)XXX-XX-XX</string>
           </property>
           <property name="clearButtonEnabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Name:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="nameLine">
           <property name="text">
            <string/>
           </property>
           <property name="maxLength">
            <number>32767</number>
           </property>
           <property name="placeholderText">
            <string>Exmpl: Ted</string>
           </property>
           <property name="clearButtonEnabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Second name:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="secondNameLine">
           <property name="placeholderText">
            <string>Exmpl: Evelin</string>
           </property>
           <property name="clearButtonEnabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Surname:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="surnameLine">
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>Exmpl: Mosbi</string>
           </property>
           <property name="clearButtonEnabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Address:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="addressLine">
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>Address</string>
           </property>
           <property name="clearButtonEnabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QGroupBox" name="genderBox">
           <property name="title">
            <string>Gender:</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QRadioButton" name="maleRadioButton">
              <property name="text">
               <string>Male</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="femaleRadioButton">
              <property name="text">
               <string>Female</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>   Date of Birth:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="dateEdit"/>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QPushButton" name="addButton">
             <property name="text">
              <string>Add</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="deleteButton">
             <property name="text">
              <string>Delete</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="changeInfoButton">
             <property name="text">
              <string>Change Information</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="infoButton">
             <property name="text">
              <string>Personal information</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QSplitter" name="splitter_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QTableView" name="tableView">
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOn</enum>
        </property>
        <property name="sortingEnabled">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QSplitter" name="splitter_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QGroupBox" name="findBox">
         <property name="title">
          <string>FindBox:</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QRadioButton" name="nameRadioButton">
            <property name="text">
             <string>Name</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="surnameRadioButton">
            <property name="text">
             <string>Surname</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="dateRadioButton">
            <property name="text">
             <string>id</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QSplitter" name="splitter_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QLineEdit" name="parametrLine">
          <property name="inputMask">
           <string/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>Name</string>
          </property>
         </widget>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QPushButton" name="findButton">
           <property name="text">
            <string>Find</string>
           </property>
          </widget>
         </widget>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>619</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuInformation">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuSave_As">
     <property name="title">
      <string>Save As</string>
     </property>
     <addaction name="separator"/>
     <addaction name="actionTXT"/>
     <addaction name="actionJSON"/>
    </widget>
    <widget class="QMenu" name="menuOpen">
     <property name="title">
      <string>Open</string>
     </property>
     <addaction name="actionTXT_2"/>
     <addaction name="actionJSON_2"/>
    </widget>
    <addaction name="menuOpen"/>
    <addaction name="separator"/>
    <addaction name="menuSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuStatistics">
    <property name="title">
     <string>Statistics</string>
    </property>
    <addaction name="actionGender_diagram"/>
    <addaction name="actionBorn_Diagram"/>
   </widget>
   <widget class="QMenu" name="menuLanguage">
    <property name="title">
     <string>Language</string>
    </property>
    <addaction name="actionEnglish"/>
    <addaction name="actionRussion"/>
   </widget>
   <addaction name="menuInformation"/>
   <addaction name="menuStatistics"/>
   <addaction name="menuLanguage"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionGender_diagram">
   <property name="text">
    <string>Gender diagram</string>
   </property>
  </action>
  <action name="actionBorn_Diagram">
   <property name="text">
    <string>Born Diagram</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionTXT">
   <property name="text">
    <string>TXT</string>
   </property>
  </action>
  <action name="actionJSON">
   <property name="text">
    <string>JSON</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionTXT_2">
   <property name="text">
    <string>TXT</string>
   </property>
  </action>
  <action name="actionJSON_2">
   <property name="text">
    <string>JSON</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>AboutQt</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionRussion">
   <property name="text">
    <string>Russian</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
